etc/generic/requirements-elyra.txt

```diff
@@ -15,6 +15,7 @@ requests==2.25.1
 tornado==6.0.4
 traitlets==4.3.3
 urllib3==1.26.5
+astunparse==1.6.3
```

Comment:
Probably dont need to change this requirements file since this is strictly for generic operators. 
Remove this dependency and adding it to the root `setup.py` should resolve the CI issue

Reply:
@akchinSTC  oh, I did put the requirement mistakenly, moved now